diagrams/py/frontend_infra_design.py

```
@@ -10,13 +10,19 @@
+    "\nFrontend Infra Design VilnaCRM",
+    show=False,
+    filename="../img/frontend_infra_design"
+):
 
     uptimerobot = Custom("\n\n\nUptimeRobot", "../src/uptimerobot.png")
     route53 = Route53("AWS Route 53")
     cloudfront = CF("AWS Cloudfront")
     lamb = Lambda("AWS Lambda")
-    lambNotifications = Lambda("AWS Lambda\n Notify object stage \n using S3 Bucket Notifies")
+    lambNotifications = Lambda(
+        "AWS Lambda\n Notify object stage \n using S3 Bucket Notifies"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)